modules/machineset-yaml-rhv.adoc

```diff
@@ -104,7 +106,9 @@ $ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
 <18> Specify the name of the secret that holds the {rh-virtualization} credentials.
 
 <19> Optional: Specify the workload type for which the instance is optimized. This value affects the `{rh-virtualization} VM` parameter. Supported values: `desktop`, `server` (default), `high_performance`. `high_performance` improves performance on the VM, but there are limitations. For example, you cannot access the VM with a graphical console. For more information see link:https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html-single/virtual_machine_management_guide/index#Configuring_High_Performance_Virtual_Machines_Templates_and_Pools[Configuring High Performance Virtual Machines, Templates, and Pools] in the _Virtual Machine Management Guide_.
-<20> A list of affinity group names that should be applied to the VMs. The affinity groups must exist in oVirt.
+<20> Optional: AutoPinningPolicy defines the policy that automatically sets CPU and NUMA settings, including pinning to the host for this instance. Supported values: `none`, `resize_and_pin`.
+<21> Optional: Hugepages is the size in KiB for defining hugepages in a VM. Supported values: 2048 or 1048576.
```

Comment:
I believe that "2048" and "1048576" should be monospace, as they're values of config parameters here.